server/app/views/applicant/NorthStarProgramIndexView.java

```diff
@@ -140,8 +140,15 @@ public String render(
             .collect(
                 ImmutableMap.toImmutableMap(
                     program -> program.programId(),
-                    program ->
-                        applicantRoutes.review(profile, applicantId, program.programId()).url()));
+                    program -> {
+                      if (profile.isPresent() && applicantId.isPresent()) {
+                        return applicantRoutes
+                            .review(profile.get(), applicantId.get(), program.programId())
+                            .url();
+                      } else {
+                        return applicantRoutes.review(program.programId()).url();
```

Comment:
In North Star, is "review" the program index page? Asking because in our current flow "review" is when you review your answers so we could definitely have an applicant at that point. But I think it's being used differently in North Star.